cmd/aperturectl/cmd/blueprints/root.go

```diff
@@ -90,7 +96,7 @@ Use this command to pull, list, remove and generate Aperture Policy resources us
 		lock = flock.New(filepath.Join(blueprintsDir, lockFilename))
 
 		// pull the latest blueprints based on skipPull and whether cmd is remove
-		if !skipPull && cmd.Use != "remove" {
+		if !skipPull && cmd.Use != "remove" && cmd.Use != "list" {
```

Comment:
On list we have to pull.

Reply:
[f201d1b](https://github.com/fluxninja/aperture/pull/1309/commits/f201d1bf88eb6c33d2940fd86578acac508ce995) addresses this